backend/src/test/java/de/neuefische/capstone/backend/controllers/MyLoansControllerTest.java

```diff
@@ -261,4 +261,73 @@ void expectUserIdNotFoundException_whenUpdateLoanWithWrongUserId() throws Except
                 .andExpect(MockMvcResultMatchers.content().json(expectedMessage));
     }
 
-}
\ No newline at end of file
+    @Test
+    @DirtiesContext
+    void expectLoanRemovedFromList_whenDeleteLoan() throws Exception {
+        UserData userDataToUpdate = new UserData("0001",
+                new ArrayList<>(List.of(
+                        new Item("1001", "€ (money)"),
+                        new Item("1002", "Book"))),
+                new ArrayList<>(List.of(
+                        new Person("2001", "Hanna"),
+                        new Person("2002", "Mona"))),
+                new ArrayList<>(List.of(
+                        new Loan("3001", "0001", "2001", "1002", "Der kleine Prinz", 1, "01.01.2023", ""),
+                        new Loan("3002", "2002", "0001", "1001", "Fahrschule", 500, "06.06.2023", "12.12.2023")))
+        );
+        UserData updatedUserData = new UserData("0001",
+                new ArrayList<>(List.of(
+                        new Item("1001", "€ (money)"),
+                        new Item("1002", "Book"))),
+                new ArrayList<>(List.of(
+                        new Person("2001", "Hanna"),
+                        new Person("2002", "Mona"))),
+                new ArrayList<>(List.of(
+                        new Loan("3001", "0001", "2001", "1002", "Der kleine Prinz", 1, "01.01.2023", "")))
+        );
+        String loanIdToDelete="3002";
+
+        myLoansRepository.save(userDataToUpdate);
+        ObjectMapper objectMapper = new ObjectMapper();
+        String updatedUserDataJson = objectMapper.writeValueAsString(updatedUserData);
+
+        mockMvc.perform(MockMvcRequestBuilders.delete("/api/myloans/"+loanIdToDelete))
+                .andExpect(MockMvcResultMatchers.status().isOk());
+
+        mockMvc.perform(MockMvcRequestBuilders.get("/api/myloans"))
+                .andExpect(MockMvcResultMatchers.status().isOk())
+                .andExpect(MockMvcResultMatchers.content().json(updatedUserDataJson));
+
+
+    }
+
+    @Test
+    @DirtiesContext
+    void expectUserIdNotFoundException_whenDeleteLoanWithWrongUserId() throws Exception {
+        String expectedMessage = """
+                {
+                 "message":"User Data not found for id: 0001"
+                }
+                """;
+        String loanId = "3001";
+        //LoanWithoutId updatedLoanWithoutId = new LoanWithoutId("0001", "2001", "1002", "Pippi Langstrumpf", 1, "01.01.2023", "");
```

Comment:
Did you leave in the comments for a reason? If not, it is best practice to delete all comments